packages/identity-service/src/routes/tiktok.js

```diff
@@ -173,32 +173,31 @@ module.exports = function (app) {
         if (isUnassociated && handlesMatch) {
           tikTokObj.blockchainUserId = userId
 
-          // if the user is verified, write to chain, otherwise skip to next step
-          if (tikTokObj.verified) {
-            const [encodedABI, contractAddress] =
-              await audiusLibsInstance.User.updateIsVerified(
-                userId,
-                config.get('userVerifierPrivateKey')
-              )
-            const senderAddress = config.get('userVerifierPublicKey')
-
-            try {
-              const txProps = {
-                contractRegistryKey: 'EntityManager',
-                contractAddress: contractAddress,
-                encodedABI: encodedABI,
-                senderAddress: senderAddress,
-                gasLimit: null
+          // Update the tiktok handle and the verified flag in DN via entity manager.
```

Comment:
How about 

Update the user's social verification status along with the social app handle that verification came from. If the user is not verified, send the transaction to ensure that their linked social handle is updated.